issue-40-built-in-benchmarking-tool.md

```diff
@@ -0,0 +1,31 @@
+## Add Built-in Benchmarking Tool
+
+## Summary
+
+There is a need to support a built-in benchmarking tool in Submariner.
+
+## Proposal
+
+Add a new ability to have a built-in benchmark tool.
+Implement a new Subctl command that will use iperf3 internally to benchmark the throughput between the two nodes.
+
+The feature Should support scheduling server and client in the same cluster.
```

Comment:
I would say that server and client can be placed within the same cluster or spread across multiple clusters.